_includes/inline-screenshot.html

```diff
@@ -1,10 +1,8 @@
 <div class="screenshot inline-screenshot pull-right">
 <a href="{{ include.source }}">
-<img src="{{ include.source }}" alt="screenshot"/>
+<img src="{{ include.source }}" {% if include.width %}style="width: {{include.width}}"{% endif %} alt="{% if include.caption %}{{ include.caption }}{% else %}screenshot{% endif %}"/>
```

Comment:
Honestly, I'm not sure why it is called "inline"; the version called inline currently floats beside the text. I'll add a property, but the terminology might be inverted from what it is currently.